app/src/main/java/com/arctouch/codechallenge/app/AppInject.kt

```diff
@@ -0,0 +1,35 @@
+package com.arctouch.codechallenge.app
+
+import com.arctouch.codechallenge.data.api.RestClient
+import com.arctouch.codechallenge.data.repository.MoviesRepository
+import com.arctouch.codechallenge.ui.base.BaseViewModel
+import com.arctouch.codechallenge.ui.details.DetailsViewModel
+import com.arctouch.codechallenge.ui.home.HomeViewModel
+import io.coroutines.cache.core.CoroutinesCache
+import org.koin.android.viewmodel.ext.koin.viewModel
+import org.koin.dsl.module.Module
+import org.koin.dsl.module.module
+
+object AppInject {
```

Comment:
Why did you choose to use Koin?

Reply:
I have been worked with Dagger a long time, and Dagger is very nice but is very verbose. Is hard to explain Dagger to a beginner colleague, on the other hand, Koin is very simple and works very well, so I have been used Koin in my new Kotlin projects.